defaults/ekscluster-kfd-v1alpha2.yaml

```diff
@@ -113,6 +113,8 @@ data:
           secretAccessKey: example
           accessKeyId: example
           bucketName: lokibucket
+        tsdbSchemav13Migration:
+          enabled: false
```

Comment:
I don't see this being used in the templates

Reply:
fixed 🩹